src/kosha/app/util.clj

```diff
@@ -1,41 +1,12 @@
 (ns kosha.app.util
-  (:require [clojure.string :as s])
-  (:import [java.util.concurrent
-            ThreadPoolExecutor
-            TimeUnit
-            ArrayBlockingQueue
-            ThreadPoolExecutor$CallerRunsPolicy]))
+  (:require [cheshire.core :as json]
+            [immuconf.config :as cfg]))
 
-;; public String unaccent(String s) {
-;;     String normalized = Normalizer.normalize(s, Normalizer.Form.NFD);
-;;     return normalized.replaceAll("[^\\p{ASCII}]", "");
-;; }
+(defn json-response [response]
+  {:status 200
+   :body (json/encode response)
+   :content-type "application/json"})
 
-(defn unaccent [s]
-  (let [normalized (java.text.Normalizer/normalize s java.text.Normalizer$Form/NFD)]
-    (.replaceAll normalized "[^\\p{ASCII}]" "")))
-
-(defn ->printable [swarams & {:keys [bold?]}]
-  (s/join ", " (map (comp s/capitalize name) swarams)))
-
-(defn display-ragam-name [ragam-name]
-  (s/capitalize (name ragam-name)))
-
-(defn create-fixed-threadpool [pool-size queue-size]
-  (ThreadPoolExecutor. pool-size  ;corePoolSize
-                       pool-size  ;maximumPoolSize
-                       60
-                       TimeUnit/SECONDS
-                       (ArrayBlockingQueue. queue-size true)
-                       ;; TODO: ask srihari why we chose this policy -nid/sd
-                       (ThreadPoolExecutor$CallerRunsPolicy.)))
-
-(defn run-in-parallel
-  "Runs tasks parallelly using a fixed threadpool and a bounded ArrayBlockingQueue,
-  where tasks are 0 argument functions."
-  [tasks pool-size queue-size]
-  (let [pool (create-fixed-threadpool pool-size queue-size)
-        rets (mapv #(.get %)
-                   (.invokeAll pool tasks))]
-    (.shutdown pool)
-    rets))
+(defonce config (cfg/load "resources/config.edn"))
+(defn get-config [& keys]
+  (apply (partial cfg/get config) keys))
```

Comment:
`apply` takes intermediate args before the list at the end. So, this can be `(apply config/get config ks)`. Use `ks` since `keys` shadows `clojure.core/keys`.